### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by rishabh486**
pr generated for alert
Screenshot (36)

### CODE REVIEW

File: alert.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+:root {
+  font-size: 62.5%;
+}
+.alert {
+  width: 30rem;
+  height: 4rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 0.3rem;
```
Comment (developer-anand) [inline alert.css:13]: you can use px for values less than 10

**Review by developer-anand — APPROVED** [review]
Good to merge with changes

File: alert.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  align-items: center;
+  margin: 0.5rem;
+
+  background-color: #209bab;
```
Comment (saurabhan) [inline alert.css]: use var to store constant values

**Review by saurabhan — APPROVED** [review]
good to merge

**Review by DivyanshSareen — APPROVED** [review]
Good to merge. would recommend using a style guide as explained in this doc https://tanaypratap.notion.site/Design-system_Guide-1b0a12ce8c084fa085e779a33169bf3b

**Review by SAGARPUJAR — APPROVED** [review]
Looks Good, please follow the Css rem and Good Code structure 
good to merge

File: alert.css (hunk 2 above)
Comment (rishabh486) [reply alert.css]: stored colors in var

File: alert.css (hunk 1 above)
Comment (rishabh486) [reply alert.css:13]: made the changes according to ur review

**Pull request merged**